ha/util/health_generator/mock_health_event_publisher.py

```diff
@@ -206,14 +206,16 @@ def get_args() -> (argparse.Namespace, argparse.ArgumentParser):
     _conf_store = ConftStoreSearch(conf_store_req=False)
 
     data_node_ids = get_data_nodes()
-    options_regex = re.compile('(-gdt$)|(-gs$)|(--get-data-nodes$)|(--get-server-nodes$)')
+    options_regex = re.compile('(-gdt$)|(-gs$)|(--get-data-nodes$)|(--get-server-nodes$)\
+                               |(publish$)|(get-disks$)|(get-cvgs$)')
 
     if len(sys.argv) > 1:
         option = sys.argv[1]
         if not options_regex.match(option):
             print(f"Error: Invalid Argument: {option}\n"
                   f"Please check supported args list:"
-                  "['-gdt|--get-data-nodes', '-gs|--get-server-nodes']")
+                  "['-gdt|--get-data-nodes', '-gs|--get-server-nodes']\n"
```

Comment:
Can we call parser_obj.print_help() here instead?

Reply:
Agree, Incorporated

Reply:
Thanks!